.eslintrc

```diff
@@ -64,7 +64,17 @@
 
```

Comment:
we can also probably update prettier to the latest version to simplify the configuration, example here: https://github.com/oneflow/kickstart-cli/pull/68/files#diff-bdf64ae6b8cf509e222acbddc72b05f1fc103f08bf70e0f5d06e4d7229e28fecR7

Reply:
good idea!